src/containers/ImportAccount.tsx

```diff
@@ -28,6 +30,10 @@ export class ImportAccountPresentation extends React.Component<
   public onSubmit = ev => {
     ev.preventDefault()
     this.props.importAccountFromSecret({ secret: this.state.privateKey })
+    sendAnalyticsEvent({
```

Comment:
See my comment below in the Sidebar index file about this triggering regardless of whether the import succeeds or fails. It's basically the same situation so I'd prefer it to be handled as per the comment below